spec/models/term_spec.rb

```diff
@@ -33,14 +33,21 @@
 
   describe "#destroy" do
     it "raises an error if the term has completed alignments" do
+      term = create(:term)
+      create(:alignment, spine_term: term, mapped_terms: [create(:term)])
+
+      expect { term.destroy }.to raise_error(RuntimeError)
```

Comment:
Would be nice to match the error message as well to make sure it's not just some random RuntimeError, but the intended one.